moveit_ros/planning/moveit_cpp/include/moveit/moveit_cpp/moveit_cpp.h

```diff
@@ -175,6 +175,10 @@ class MoveItCpp
   execute(const std::string& group_name, const robot_trajectory::RobotTrajectoryPtr& robot_trajectory,
           bool blocking = true, const std::vector<std::string>& controllers = std::vector<std::string>());
 
+  [[deprecated("MoveItCpp::execute() no longer requires a blocking parameter")]] moveit_controller_manager::ExecutionStatus
```

Comment:
Your rebase was not performed correctly because this is removed in the main branch. Please clean this up, you should only see the diffs you created here.

Reply:
Oh I see, it was swapped with the last pull request I made in this repo. It'll be fixed soon, I apologize for the delay.